flink-simple-tutorial/README.md

```diff
@@ -87,9 +87,9 @@ The `HeapMetrics` class has a few key properties:
 2. It has a public empty constructor
 3. All fields are public non-final
 
-With these properties, the class is made to be efficiently serializable by the Flink type system.
+These classes are called POJOs (Plain Old Java Objects) in the Flink community. They are supported out of the box by Flink for serialization. As Flink is a distributed system transferring data between machines is a fundamental functionality and a potential performance bottleneck. 
```

Comment:
As Flink is a distributed system, transferring data between machines is a fundamental functionality and a potential performance bottleneck.